vm/ubpf_jit_arm64.c

```diff
@@ -1189,6 +1189,10 @@ translate(struct ubpf_vm* vm, struct jit_state* state, char** errmsg)
                 *errmsg = ubpf_error("Too many LEA calculations.");
                 break;
             }
+            case TooManyLocalCalls: {
+                *errmsg = ubpf_error("Too many local calls.");
```

Comment:
Just talking out loud: There is no need to do an equivalent jump around trick on ARM because the return address is not pushed onto the stack -- it is kept in the branch/link register. As a result, the stack does not get misaligned on a call like it does on x86. 

Do you agree with that?

Reply:
Unfortunately, no. ARM64 now also has a prolog, so we now need to do the same.